src/Mod/Arch/exportIFC.py

```diff
@@ -55,8 +55,8 @@
 __url__    = "https://www.freecad.org"
 
 # Save the Python open function because it will be redefined
-if open.__module__ in ['__builtin__', 'io']:
-    pyopen = open
+if open.__module__ in ('__builtin__', 'io', '_io'):
+
```

Comment:
```suggestion
from builtins import open as pyopen
```

Reply:
Ok, I did both and pushed. Hopefully (we'll see in the CI) there are no more python errors.